charts/op-scim-bridge/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-name: op-scim
-version: 2.4.2
+name: op-scim-bridge
+version: 2.5.0
```

Comment:
This version also needs to be set in `appVersion`.

Reply:
The `version` and `appVersion` are distinct and don't need to be linked like they were before. We use the `appVersion` to set the `scim.version` in `values.yaml`:
```yaml
scim:
  # ...
  # version of the SCIM bridge
  version: "{{ .Chart.AppVersion }}"
```

And then in turn we use the `scim.version` to set the image tag version for the Docker image in the `deployment.yaml`:
```yaml
      containers:
        - name: {{ tpl .Values.scim.name . }}
          image: {{ .Values.scim.imageRepository }}:{{ tpl .Values.scim.version . }}
```

We don't yet have a version `2.5.0` tagged on the Docker image.

With the latest release of this chart (2.4.2), and going forward, we no longer need to keep the chart version and SCIM bridge application (tag) version in lockstep with each other.